common/cpu/amd/pstate.nix

```diff
@@ -0,0 +1,10 @@
+{ lib, config, ... }: {
+  #Enables the amd cpu scaling https://www.kernel.org/doc/html/latest/admin-guide/pm/amd-pstate.html
+  #On recent AMD cupus this can be more energy efficient
```

Comment:
```suggestion
  # Enables the amd cpu scaling https://www.kernel.org/doc/html/latest/admin-guide/pm/amd-pstate.html
  # On recent AMD cupus this can be more energy efficient
```